Mathlib/Algebra/Order/Kleene.lean

```diff
@@ -133,6 +133,11 @@ scoped[Computability] attribute [simp] add_eq_sup
 
 theorem add_idem (a : α) : a + a = a := by simp
 
+lemma cast_eq_one {n : ℕ} (nezero : n ≠ 0) : (n : α) = 1 := by
```

Comment:
Can you add the `ofNat` version of this too, which should be proved directly with this lemma?

Reply:
Done, thanks!